server/askserver/askserver/tests/mockData.py

```diff
@@ -0,0 +1,42 @@
+from django.utils import timezone
+from askserver.models import User, Tag, Thread, Message, Report
+import uuid
+
+users = [
+    User(uid="user1", displayname="John Doe", pronouns="he/him", permissions=1),
+    User(uid="user2", displayname="Jane Smith", pronouns="she/her", permissions=4),
+    User(uid="user3", displayname="Alex Johnson", pronouns="they/them", permissions=2),
+]
+
+User.objects.bulk_create(users)
+
+tags = [
+    Tag(name="Calc"),
+    Tag(name="CCI"),
+    Tag(name="API"),
+]
+
+Tag.objects.bulk_create(tags)
+
+threads = [
+    Thread(title="How to find the derivative of...", author=users[0], date=timezone.now()),
```

Comment:
> `author=users[0]`

How would this actually look in an API call (i.e. one to the database)?

Reply:
author=users is code I initially wrote using Django's default/built in user authentication. I had some issues with it because It was not similar enough to our project so I refactored to user = settings.AUTH_USER_MODEL, and this piece of code has not been updated yet. But the reason its an array of users was to test different permission settings.